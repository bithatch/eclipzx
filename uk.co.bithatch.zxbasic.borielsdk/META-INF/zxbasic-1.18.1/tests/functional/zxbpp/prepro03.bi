
#define test x 'a comment \
    this line must be compiled
#define test2 y ' This define should not be ignored

test
test2
