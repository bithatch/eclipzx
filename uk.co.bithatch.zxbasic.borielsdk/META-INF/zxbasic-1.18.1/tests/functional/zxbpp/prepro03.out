#line 1 "prepro03.bi"

#line 4 "prepro03.bi"
#line 5 "prepro03.bi"

x 
    this line must be compiled
#line 7
y
